src/Compositor.cpp

```diff
@@ -2339,7 +2343,6 @@ void CCompositor::setWindowFullscreenState(const PHLWINDOW PWINDOW, sFullscreenS
     EMIT_HOOK_EVENT("fullscreen", PWINDOW);
 
     PWINDOW->updateDynamicRules();
-    PWINDOW->updateWindowDecos();
```

Comment:
why was this removed?

Reply:
`updateWindowAnimatedDecorationValues(PWINDOW)` calls `updateWindowDecos()`.

https://github.com/hyprwm/Hyprland/blob/bdb296a83c595f3814f8159d2cffd25c3edfb842/src/Compositor.cpp#L1941